h/static/scripts/annotator/config.js

```diff
@@ -21,9 +35,9 @@ function config(window_) {
     }
   }
 
-  var annotFragmentMatch = window_.location.hash.match(/^#annotations:(.*)/);
-  if (annotFragmentMatch) {
-    options.annotations = annotFragmentMatch[1];
+  var directLinkedID = annotationIDs.extractIDFromURL(window_.location.href);
```

Comment:
Ah, now that you mention "last resort", that makes me realize that in the context of Via, we could re-use the mechanism added here for the Chrome extension and have Via inject the same config tag into the page.